frontend/src/store/reducers/userReducers.js

```diff
@@ -0,0 +1,70 @@
+import {
+	REGISTER_USER,
+	LOGIN_USER,
+	LOGOUT_USER,
+	FETCH_USER_DATA,
+	USER_DATA_FETCHED,
+	ERROR
+} from '../actions';
+
+const initialState = {
+	users: [],
+	fetching: false,
+	success: false,
+	error: null
```

Comment:
Using null here gets a bit dicey when using prop-types, because prop-types doesn't have a null or undefined property check. Consider an empty string perhaps?